lib/pgbackups-archive/job.rb

```diff
@@ -94,12 +92,12 @@ def file
   end
 
   def key
-    timestamp = created_at.gsub(/\/|\:|\.|\s/, "-").concat(".dump")
-    ["pgbackups", environment, timestamp].compact.join("/")
+    timestamp = created_at.gsub(/\/|\:|\.|\s/, "-").gsub(/\+/, '').concat(".dump")
```

Comment:
Good call on removing the `+`.  I'm wondering if we should just remove the entire "-+0000" though.